recipes/mudirac/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+mkdir -p build
+cd build
+cmake $SRC_DIR
+make
+ctest
+mkdir -p $PREFIX/bin/
+cp -r bin/mudirac $PREFIX/bin/
```

Comment:
Doesn't this have a `make install` command?

Reply:
it doesnt have `make install` its either `make mudirac` which only builds the package and skip tests and etc.